javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,68 @@
+// Flight booking fullname function
+
+const useFormalName = "Lord" + getFullname;
+
+getFullname("Benjamin", "Hughes", useFormalName);
+let fullname1  = "Benjamin";
+let fullname2  = "Hughes";
+function getFullname(fullname1, fullname2) {
+  if (useFormalName) {
+  return ("Lord" + " " + fullname1 + " " + fullname2)
+} else {
+  return (fullname1 + " " + fullname2)
+}
+}
+getFullname(fullname1, fullname2);
+/* the getFullname for women will work the similar way: we need to create boolean variable useWomenName and if it's true
+we should use Queen as a formal way instead of Lord.
+*/
+
+// Event application
+function getEventWeekday(){
+let days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]
+const todaysDate = new Date();
+
+}
+// Weather wear
+
+function whatToWear(a) {
+  if (a >= 0 && a <= 15) {
+    console.log("Wear jeans and a jacket")
+  } else if (a >= 16) {
+    console.log("Wear top and shirts")
+  } else {
+    console.log("Wear a space suit - it is too cold!")
+  }
+}
```

Comment:
Well done! Nice and simple, understandable function! ⚡ Just make sure not to `console.log()` in the function, instead, retur the string, so you can store the value in the `clothesToWear` variable. 👍